src/components/atoms/CyToggle.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <label class="cy-toggle">
+    <input type="checkbox" />
+    <span class="slider round"> </span>
+  </label>
+</template>
+<script>
+export default {
+  name: 'CyToggle',
```

Comment:
- CyToggle should accept a prop called `value` which is to be a _boolean_ 
- CyToggle should emit an event called "input" like so: `this.$emit('input', !this.value)`

these two conditions will make CyToggle usable with `v-model` inside Vue components wherever we use CyToggle. 😉